docs/content/doc/upgrade/from-gogs.en-us.md

```diff
@@ -70,9 +70,14 @@ There are some basic steps to follow. On a Linux system run as the Gogs user:
 
 ## Upgrading to most recent `gitea` version
 
-After successful migration from `gogs` to `gitea 1.0.x`, it is possible to upgrade to the recent `gitea` version.
-Simply download the file matching the destination platform from the [downloads page](https://dl.gitea.io/gitea)
-and replace the binary.
+After successful migration from `gogs` to `gitea 1.0.x`, it is possible to upgrade `gitea` to a modern version
+in a two steps process.
+
+Upgrade to [`gitea 1.6.4`](https://dl.gitea.io/gitea/1.6.4/) first. Download the file matching
+the destination platform from the [downloads page](https://dl.gitea.io/gitea/1.6.4/) and replace the binary.
+Run Gitea at least once and check that everything works as expected.
+
+Then repeat the procedure, but this time using the [lastest release](https://dl.gitea.io/gitea/).
```

Comment:
Can you use the `{{< version >}}` variable maybe?  
Or perhaps point them to the FAQ for versions, since we have many questions about the download site in regards to versions and what "latest" means.

Reply:
@jolheiser Done. 😅